### DISCUSSION TITLE
Can't see text in Spotify app Search field

### DISCUSSION THREAD

**Discussion by alienfxfiend** [SpotX-Official/SpotX#82] [General]
I cannot see the text in the Spotify Search [in the app], it's literally black text on black background. This has cropped up recently, I was able to type in the Search clearly before ! If I press Ctrl+A I can see the text highlighted again. P.S: I reinstalled Spotify w/o SpotX, & the search text is still invisible —seems to be an issue /w Spotify app, hope they fix it ! P.P.S: Someone had tampered /w my WindowBlinds 10 skin, that's why the Search field was black on black. It's fixed now !

**Comment by alienfxfiend** [comment]
Is there a quick way to make my Search field in Spotify app White /w black text ? 'Cause my WindowBlinds black+navy blue skin is 'causing it to be dark & the text is invisible. This cropped up recently.

**Reply by amd64fox** [reply]
For you, this can only be fixed through manual css editing, where you can set any color for this field, but you need to understand that with the next update, all this will disappear.

**Comment by alienfxfiend** [comment]
Thanks, I was able to use Spicetify (default theme) so it's visible nicely & I've retained my WindowBlinds skin ! I realize the update overwrites all files, I will just have to re-apply it. Also SpotX is awesome !

**Reply by amd64fox** [reply]
Oh yeah, I forgot about Spicetify.
Also, to avoid the upcoming update, you can simply block it.

**Reply by alienfxfiend** [reply]
Like the README said I ran SpotX (full .BAT) after Spicetify's [update] command, but I get the errors: "SpotX has already been installed, xpui.js and xpui.css not found." Will this affect the adblock functionality in future patches 😧 ? ('cause of the missing files.) Because I had run & patched Spotify /w SpotX before I actually used Spicetify […] I ask this because the Enhance button is still active.

**Comment by alienfxfiend** [comment]
Could you please clarify ? ^^

**Reply by amd64fox** [reply]
The patch does not find backup files to restore, which is why this error appears
Just uninstall Spotify and Spicetify and reinstall everything.

I recommend installing SpotX first and then Spicetify

**Reply by alienfxfiend** [reply]
> The patch does not find backup files to restore, which is why this error appears Just uninstall Spotify and Spicetify and reinstall everything.
> 
> I recommend installing SpotX first and then Spicetify

Awesomesauce ! Yeah that's what I did !
